spec/models/user_spec.rb

```diff
@@ -4,6 +4,15 @@
   describe "validations" do
     it { should validate_presence_of(:name) }
     it { should validate_presence_of(:email) }
+
+    it "should not allow an email to be changed" do
+      user = create(:administrator)
```

Comment:
Is it worth explicitly doing `user = create(:administrator, email: "old@example.com")` ?